frontend/src/pages/results.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useRef, useState } from "react";
+import Chart from "chart.js";
+import Button from "react-bootstrap/Button";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Grid from "@material-ui/core/grid";
+import Stepper from "./stepper.js";
+import Radar from "./chart/chart";
+import EMenu from "./notesMenu.js";
+import { Typography, Box } from "@material-ui/core";
+
+function Results() {
+  let Summary_Value = 2.03;
+  let Coverage = { Safety: 0.5, Salary: 0.667, Reputation: 1.0, Privacy: 0.8 };
+  return (
+    <div>
+      <Grid direction="row" justify="center" alignItems="center">
+        <Typography variant="h4" align="center" gutterBottom="true">
```

Comment:
I think there should maybe be a line break between this typography and the navbar. They are a bit close atm